build_defs/stamp_plugin_xml.py

```diff
@@ -203,7 +203,7 @@ def main():
   for new_element in new_elements:
     idea_plugin.appendChild(new_element)
 
-  print dom.toxml(encoding="utf-8")
+  print(dom.toxml())
```

Comment:
FYI the encoding was there for a reason, please don't remove it. It caused errors when we didn't have it, I believe because we have non-ASCII characters in our changelogs.

The docs for 2.7 specifically say you should ALWAYS include it: https://docs.python.org/2.7/library/xml.dom.minidom.html#xml.dom.minidom.Node.toxml

Reply:
No problem. I had to remove this, because `dom.toxml(encoding="utf-8")` returns a byte-string, but `print` only accepts text-strings.

I'll add a commit that should restore the previous behavior.

Reply:
FYI, in `merge_xml.py` you already had a plain `print(dom.toxml())` without a specified encoding - shall we keep this as is, or should I add a drive-by fix for that file as well? :)